gulpfile.js

```diff
@@ -1,78 +1,78 @@
-var gulp = require('gulp');
-var zip = require('gulp-zip');
-var exec  = require('exec-chainable');
-var eslint = require('gulp-eslint');
-var path = require('path');
-process.env.ARTEFACT_PATH = __dirname;
-var riffraff = require('node-riffraff-artefact');
+var gulp = require('gulp')
+var zip = require('gulp-zip')
+var exec = require('exec-chainable')
+var eslint = require('gulp-eslint')
+var path = require('path')
+process.env.ARTEFACT_PATH = __dirname
+var riffraff = require('node-riffraff-artefact')
 
-var LAMBDA_SOURCE = 'src/*.js';
-var DEPLOY_SOURCE = 'conf/deploy.json';
-var CLOUDFORMATION_SOURCE = 'conf/cloudformation/*.yml';
+var LAMBDA_SOURCE = 'src/*.js'
+var DEPLOY_SOURCE = 'conf/deploy.yml'
```

Comment:
Ok, no worries. I will reapply the changes from the master branch version of `gulpfile.js` directly.